### PR TITLE
Fix issue #3 errors & a lot of misc. errors

### PR DESCRIPTION
**Pull request by jasonlewcodes**
🙂

### CODE REVIEW

**Comment by Joe-McCann** [comment]
### <span aria-hidden="true">👷</span> Deploy Preview for *joe-mccann* processing.


|  Name | Link |
|---------------------------------|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 7b63e7571ca765683f7b5c2a4e6afc539da983b7 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/joe-mccann/deploys/63d297da32d38a000853ddbf |

File: content/course/introToLogic/Sections/Propositions.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ its a good idea to perform yearly inspections, or you trying to lie to your mom
 Compared to our previous math example, these examples are based on the idea of adding propositions into
 our sentences and arguments. A proposition that has a linguistic meaning because of words used inside of
 it are called __statements__; note that the proposition here is the true or false evaluation of our statement
-based on its meaning. This is all much more philisophical than our monke applied brains really care for,
```
Comment (Joe-McCann) [inline content/course/introToLogic/Sections/Propositions.md:46]: monke was intentionally mispelled here per meme

**Review by Joe-McCann — CHANGES_REQUESTED** [review]
Revet the one `monke` change and I will approve. Great changes, I seem to misspell things a lot

File: content/course/introToLogic/Sections/Propositions.md (hunk 1 above)
Comment (jasonlewcodes) [reply content/course/introToLogic/Sections/Propositions.md:46]: 👍

**Review by Joe-McCann — APPROVED** [review]
Looks good, thank you

**Pull request merged**